### PR TITLE
Added black linter workflow

### LINKED ISSUE #307: [Feature] Add black linter workflow
## Are you adding a new feature?
- [x] YES
- [ ] NO

# Are you enhancing the old feature?
 - [ ] YES
 - [x] NO

 ## Is your feature request related to a problem?
 - [x] YES
 - [ ] NO

## Describe your feature/enhancement.

Add the black linter workflow to ensure proper linting of the code. As a project with multiple users contributing to it, it is essential to maintain a proper linting style to ensure better code readability by users. An excellent way to do this is by automatic linting checks using GitHub actions.

I would like to work on this project as a part of GSSoC 23. Please assign this issue to me.

## Can you add the feature by yourself?

- [x] YES
- [ ] NO

### PR DESCRIPTION
**Pull request by metabiswadeep**
Fixes #307 

Added the black linter workflow using Github actions for the repository

### CODE REVIEW

File: .github/workflows/black.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    runs-on: ubuntu-latest
+
+    steps:
+
+      - uses: actions/checkout@v3
+
+      - name: Format code with black
+        run: |
+          pip install black
+          black . --exclude "main.py"
```
Comment (AshAman999) [inline .github/workflows/black.yml:20]: May I know why we excluded the main.py file here >?

@metabiswadeep

Comment (metabiswadeep) [reply .github/workflows/black.yml:20]: The main.py file has lines beginning with @app.get which the black linter cannot parse due to it not being Python code, and raises errors.

**Comment by metabiswadeep** [comment]
@ighoshsubho Any modifications necessary for this?

**Comment by ighoshsubho** [comment]
No, will be merging this by night...

**Comment by ighoshsubho** [comment]
Congrats @metabiswadeep, your PR got merged! Star this repo and keep contributing like this in GSSOC'23

**Pull request merged**